monash-human-power/dashboard
# Kdav108/azure deployment

## Description

Adds a GitHub Action workflow to push changes merged into master to the Azure deployment of dashboard.

## Screenshots

N/A

## Steps to Test

1. Merge code into master
2. Check the status of the workflow/action that is triggered form the Actions tab in GitHub.
3. Check live website to see if the changes are live!
